Pilot/LoginViewController.swift

```diff
@@ -81,17 +81,21 @@ class LoginViewController: NSViewController {
     userService.getPilotUser(usernameTextField.stringValue, password: hashedPassword,
       completion: { user in
 
-        // User is correct so try to store the password in keychain
-        do {
-          try Locksmith.saveData(["password": user.password], forUserAccount: user.username)
-        } catch {
-          // Could not load account into keychain
-          print("Couldn't load the password to keychain")
-        }
+//        // User is correct so try to store the password in keychain
```

Comment:
If we're leaving this commented out, put a comment explaining why it's commented out for now.